src/tests/e2e/pages/emailTransferForm.page.js

```diff
@@ -0,0 +1,125 @@
+import log from 'loglevel'
+import { getWalletTitle } from '../../../wallet.js'
```

Comment:
don't we need to add moduleNameMapper in jest.config.js to make this work?

Reply:
That part has been merged.